packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -460,15 +462,32 @@ export const Webchat = forwardRef((props, ref) => {
                 onKeyDown={e => onKeyDown(e)}
                 style={{
                   display: 'flex',
-                  paddingLeft: '10px',
                   fontSize: 14,
                   border: 'none',
                   resize: 'none',
                   overflow: 'auto',
                   outline: 'none',
-                  marginTop: '13px'
+                  flex: '1 1 auto',
+                  padding: '14px 10px 0px 0px'
                 }}
               />
+              {props.emojiPicker && (
```

Comment:
this method is too large.
Please create in this PR a method like renderEmojoPicker with this code, and in another PR try to split this huge return in smaller chunks. 
thanks

Reply:
what method? this is not a method. It's only a condition for seeing if user has declared emojiPicker in his index.js file

Reply:
I meant that the function it belongs to is too long. Please Move lines 475 to 490 to a different method in the is PR. In another PR, please do the same to simplify further the Webchat return (from line 336 till the end of the file).
Ideally it should look st like:
``` return (
    <WebchatContext.Provider
      value={{
        sendText,
        sendPayload,
        openWebview,
        resolveCase,
        webchatState,
        addMessage,
        toggleWebchat,
        updateMessage,
        updateReplies,
        staticAssetsUrl
      }}
    >
   {renderTriggerButton(webChatState)}
   {webchatState.isWebchatOpen && (
        <div
          style=divStyle(webchatState)
        >
         header(toggleWebchat)
         messageList(webchatState)
         replies(webchatState)
         emojiIsOpened && emojiPicker()
         persistenMenu(...)
        ....
```